qutebrowser/config/configdata.yml

```diff
@@ -758,6 +758,23 @@ content.notifications:
     QtWebKit: true
   desc: Allow websites to show notifications.
 
+content.notification_presenter:
+  default: libnotify
+  restart: true  # no way to clear the presenter libnotify installs
```

Comment:
Calling `setNotificationPresenter(None)` won't work?

Reply:
It just tries to call `None`. The underlying C++ API takes a function call, so I think this is a Qt limitation.

Reply:
Huh, odd. PyQt has an `AllowNone` annotation for it. From what I can see in Qt, setting it to `None` / `nullptr` will just ignore notifications entirely rather than using the default Qt presenter - is that what you've seen?